spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -258,6 +258,9 @@ A portable extension or other object may obtain a reference to the current conta
 Other methods on `CDI` may be called after the application initialization is completed until the application shutdown starts.
 If methods on `CDI` are called at any other time, non-portable behavior results.
 
+When performing lookup of bean instance with `CDI` the qualifiers specified in the `Instance.select()` method are used to perform typesafe resolution.
```

Comment:
For `javax.enterprise.inject.spi.CDI` I would start more generally, e.g.:

`CDI` implements `javax.enterprise.inject.Instance` and therefore might be used to perform programmatic lookup.
If no qualifier is passed to the `Instance.select()` method the `@Default` qualifier is assumed.